docs/content/en/docs-dev/user-guide/managing-controlplane/configuration-reference.md

```diff
@@ -147,6 +147,7 @@ Must be one of the following objects:
 |-|-|-|-|
 | name | string | The unique name of the configuration. | Yes |
 | provider | string | The SSO service provider. Can be one of the following values<br>`GITHUB`, `GOOGLE`... | Yes |
+| sessionTtl | int | The time to live of session for SSO login. Unit is `minute`. | Yes |
```

Comment:
Do we have any reason to mark this field as required? We have a default value for this setting anyway.